pages/manage.js

```diff
@@ -0,0 +1,52 @@
+import { useRouter } from 'next/router'
+import Select from 'react-select'
+import { useAuth } from 'use-auth0-hooks'
+
+import UserList from '../components/UserList'
+import { AUTH0_SCOPE } from '../util/constants'
+
+export default function Manage () {
+  const { isAuthenticated, isLoading } = useAuth({
+    audience: process.env.AUTH0_AUDIENCE,
+    scope: AUTH0_SCOPE
+  })
+  const { push, query: { type } } = useRouter()
+  if (!isLoading && !isAuthenticated) {
+    return (
+      <div>
+        Please log in to view the Admin Dashboard.
+      </div>
+    )
+  }
+  const manageOptions = [
+    // TODO: Remove Home or do we want a summary view here?
+    {label: 'Home'}, // value is undefined to match missing query param
+    {value: 'admin', label: 'Admin Users'},
+    {value: 'otp', label: 'OpenTripPlanner Users'}
+  ]
+  return (
+    <div>
+      <h1>Manage</h1>
+      <Select
```

Comment:
I'd much rather have tabs or some kind of navbar instead of a dropdown.

Reply:
See #7 for tracking further refinements. I think gathering feedback on specific UI elements/approaches is fine, but perhaps we should keep these separated from the PR review.